src/composables/popupProps.ts

```diff
@@ -22,7 +23,7 @@ export function usePopupProps(): IPopupPropsState {
 
   const sender = computed((): IAccountOverview => ({
     name: popupProps.value?.app?.name,
-    address: popupProps.value?.app?.host,
+    address: popupProps.value?.app?.host as Encoded.AccountAddress,
```

Comment:
Actually this is not correct. The `host` is an URL. This is like saying that `cat` is a `chair` :D